src/Cdn/custom/Remove-AzFrontDoorCdnProfile.ps1

```diff
@@ -155,21 +155,36 @@ function Remove-AzFrontDoorCdnProfile {
     )
 
     process {
+        $hasAsJob = $PSBoundParameters.Remove('AsJob')
+        $hasNoWait = $PSBoundParameters.Remove('NoWait')
+        $hasPassThru = $PSBoundParameters.Remove('PassThru')
```

Comment:
Specify `PassThru` in Remove-AzFrontDoorCdnProfile.Tests.ps1

Reply:
Same to Remove-AzFrontDoorCdnProfile.Tests.ps1, `PassThru` has added as a single testcase 
https://github.com/TracyYaoYao/azure-powershell/blob/2a213ccb06040ef7f38fbf1d264899831c018fba/src/Cdn/test/Remove-AzFrontDoorCdnProfile.Tests.ps1#LL32C7-L32C7